conf/citest/pipeline/LoadMember.json

```diff
@@ -0,0 +1,60 @@
+{"Pipeline_citest" : {
+  "pipeline_name":"load_member",
+  "tables":[
+    {
+      "name":"dnafrag",
+      "variation":0,
+      "lst_group_by":[],
+      "columns":[{"name":"*"}]
+     },
+    {
+      "name":"gene_member",
+      "variation":0,
+      "lst_group_by":[],
+      "columns":[{"name":"*"},{"name":"canonical_member_id"}]
```

Comment:
Why make this an array of dictionaries, rather than just an array? What attributes (other than `name`) do you see being included later?